spec/spec_helper.rb

```diff
@@ -2,20 +2,30 @@
 Bundler.setup(:default, :development)
 require 'capybara/rspec'
 require 'capybara-screenshot/rspec'
+require 'pry'
 
 Capybara.default_driver = :selenium
-
 Capybara.save_path = File.expand_path('../screenshots', __dir__)
 Capybara.default_max_wait_time = 10
+
 RSpec.configure do |config|
   config.disable_monkey_patching!
   config.before(:all) do
-    # page.driver.browser.manage.window.resize_to(1900, 1200)
+    check_firefox_version
```

Comment:
Why would you we want to check the Firefox version?
Tests will fail if the FF version isn't compatible anyway.

Reply:
@StupidCodeFactory - this is related to a bug I have experienced on other projects. For some reason the click_button method does not appear to work in more recent versions of Firefox and it's not obvious that the browser version is the issue. This is why the README's for prison-visits-public and prison-visits-staff specify a certain version of Firefox. Open to suggestions / alternatives though. Currently, Circle CI is using Firefox v56

Reply:
Yes I have experienced that bug as well (usually do a page.find('button').click to fix that If I recall correctly)
I'd rather have it be a warning rather than fail directly, we'll just find another to click on the button if this happens.